admindb/interface.go

```diff
@@ -1,5 +1,13 @@
 // SPDX-License-Identifier: MIT
 
+// Package admindb implements all the persisted database needs of the room server.
```

Comment:
One thing I kept thinking: since this package takes care of persisting all the data, then maybe `admindb` isn't really descriptive? `roomdb` might be better; `admindb` implies being concerned with a database of admins, I think, suggesting the presence of other databases as well. 

I didn't really want to comment on this though since it seems so nitpicky, but I kept thinking about it so 🤷

Reply:
:+1: 

I had a similar thought when I wrote this but didn't want to overboard this PR. I will do it in a follow up similar to #53

Reply:
cool, glad to hear it validated something you were thinking of! :) i was really hesitant to comment on it haha

Reply:
I created #57, too :)